google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -767,3 +769,27 @@ def export_images(
             index += 1
 
         return output_filenames
+
+    def export_hocr_str(self, title: str) -> str:
+        r"""Exports a string hOCR version of the Document.
+
+        Args:
+            title (str):
+                Required. The title for hocr_page and head.
+
+        Returns:
+            str:
+                A string hOCR version of the Document
+        """
+        environment = Environment(loader=FileSystemLoader("templates/"))
+        template = environment.get_template("hocr_xml_template.txt")
+        hocr_pages = ""
+        number_of_pages = len(self.pages)
+        for page_to_export in self.pages:
+            hocr_pages += page_to_export.to_hocr(title=title)
+
```

Comment:
This seems to say that the same title is shared across the whole document and every page?

In the hOCR specification must each page have a title?